src/MoBi.Assets/AppConstants.cs

```diff
@@ -88,6 +88,7 @@ public static class Organs
       public static class DefaultNames
       {
          public static readonly string MoleculeBuildingBlock = "Molecules";
+         public static readonly string Module1 = "Module1";
```

Comment:
module1?

Reply:
The variable should at least be called Module not Module1. Thjis is not the winform designer here